testing/test_transfer_learning.py

```diff
@@ -1,23 +1,24 @@
-
 import torch
 import torch.nn as nn
 import torch.backends.cudnn as cudnn
 from torch import optim
 
-from ivadomed import models
+import copy
+
+from ivadomed import models as imed_models
 
 cudnn.benchmark = True
 
-GPU_NUMBER = 0
```

Comment:
I remember seeing this comment but... do we need to standardize this? In this case: I simply changed it because when I was changing the GPU 0 was full busy. It may have elegant solutions to handle this.

Reply:
Yes that would be a great idea! Personally, I have only 1 gpu on my computer, so putting 7 as GPU_NUMBER triggers an error, so I'd be in favor to have 0 by default and if it is used choose another free GPU seems like a great idea :D!

Reply:
Note: this test does not need a GPU --> could be removed.

Reply:
Happy to put 0 as default :)